events/controller.js

```diff
@@ -0,0 +1,24 @@
+const { raw } = require('objection');
+const EventOccurrencesModel = require('./models/EventOccurrencesModel');
+const QueryHelper = require('../util/QueryHelper');
+
+
+class EventsController {
+  static async list(query) {
+    const queryHelper = new QueryHelper(EventOccurrencesModel, query);
+    const qb = queryHelper.qb;
+    qb.distinct(raw('ON (start_time) *'));
+
+    if (query.after_date) {
```

Comment:
use these after_date as a computed field in the model to avoid having to do this for every endpoint?

Reply:
👍